src/web/views/Components/SearchAutocomplete/props-data.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable */
+export default [
+  [ "Props", "Type", "Default value", "Description" ],
+  [
+    "items",
+    "Array",
+    "() => []",
```

Comment:
Here we can only put `[]` as default value.
The arrow function that return an empty array is the way we tell Vue the way it can get the default value for this prop. But the default value is not the actual function but its return value.